src/app/modules/layout/components/admin-layout/admin-layout.component.ts

```diff
@@ -209,4 +151,13 @@ export class AdminLayoutComponent implements OnInit, AfterViewInit, OnDestroy {
         }
       });
   }
+
+  private listenToRouteChange(): void {
```

Comment:
This can also be setup inside the service instead of leaving it up to the component. Maybe service can emit an observable on when to close the navbar at most but the decision making should happen inside the service itself.